README.md

```diff
@@ -1 +1,139 @@
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+# 🛠 Built With
+
+- HTML5
+- CSS
+- Javascript ES6
+
+# Key Features
+
+- Add and remove books
+
+  - The website that displays a list of books and allows you to add and remove books from that list.
+
+- JavaScript Classes
+
+  - The books are managed and manipulated using classes and helper functions within those classes.
+
+- Store and retrieve data
+  - The displayed books are stored and retrieved from the local storage of the browser.
+
+# 🔭 Future Features
+
+> I will be adding More functions.
+
+## Learning Objectives
+
+- Flexbox to place elements in the page.
+- Use javascript objects to save data.
+- Practice of all the knowledge i learned in this module.
+
+# 📖 The webster (First capstone project)
+
+<a name="about-project"></a>
+
+> This is a website for an online book library.
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **It's a website according to the design provided**
+- **It allows user to add or delete books from a book library**
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+- [Live Demo Link](https://github.com/DannieMo/Awesome-Books-ES6.git)
+
+### Prerequisites
+
+In order to run this project you need:
+
+### Setup
+
+Clone this repository to your desired folder:
+
+**Just clone the project by runing the `git clone git@github.com:DannieMo/Awesome-Books-ES6.git` command in your command line, and you are done.**
+
+<!-- AUTHORS -->
+
+## Authors
+
+👤 **Daniel Majesty Owiri (Dannie)**
+
+- GitHub: [@dannieMo](https://github.com/DannieMo/)
+- LinkedIn: [Daniel Majesty Owiri](https://linkedin.com/in/daniel-majesty-owiri/)
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project...
+
+Give a ⭐️ if you like my effort!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank the whole Microverse community for their help and contributions towards this project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE)
+
```

Comment:
- [ ] This link is not working because you don't have a license file in your application :fearful: Kindly add a license file using these [guidelines](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) :+1:  make your application licensed which makes you and your application professional :blush: